ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -126,6 +126,13 @@ export const Alleles = React.memo(({ genotype, variant, isHemiX, warning }) =>
         content={<div><b>Warning:</b> {warning}</div>}
       />
     }
+    {cnSvtypeMissmatch &&
```

Comment:
we don't want a second icon popup here, we want to use the `warning` prop for this